website/docs/plugins/submitit_launcher.md

```diff
@@ -29,45 +29,59 @@ defaults:
 Note that this plugin expects a valid environment in the target host. usually this means a shared file system between
 the launching host and the target host.
 
-Submitit supports 3 kind of queues: auto, local and slurm. Its config looks like this
+Submitit supports 3 types of queues: auto, local and slurm. Its config looks like this
```

Comment:
Yeah, I am starting to think that maybe yaml is a better form to document the config despite it missing the types.

Reply:
agreed